tests/cli/workflow/nf_analysis/test_cli_compound_commands.py

```diff
@@ -118,3 +118,74 @@ def test_store_fail(
 
     # THEN bundle exist status should be
     assert result_fail.exit_code != EXIT_SUCCESS
+
+
+@pytest.mark.parametrize(
+    "workflow",
+    [Workflow.RNAFUSION, Workflow.TAXPROFILER],
+)
+def test_store_available(
+    cli_runner: CliRunner,
+    workflow: Workflow,
+    real_housekeeper_api: HousekeeperAPI,
+    deliverables_template_content: list[dict],
+    request: FixtureRequest,
+    caplog: LogCaptureFixture,
+    mocker,
+):
+    """Test to ensure all parts of compound store-available command are executed given ideal conditions
```

Comment:
it would be nice to add a case that fails to store and one case that completes to make sure the commands doesn't stop after failing one case

Reply:
Added now.